.buildkite/build_pr_pipeline.yml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: ":white_check_mark: Build docs PR"
+    command: |
+      bash .buildkite/scripts/build_docs_pr.sh
+    agents:
+      provider: "gcp"
+      image: family/docs-ubuntu-2204
+notify:
+  - email: "docs-status@elastic.co"
```

Comment:
I don't think we ever want to notify via email on failed PRs (especially to content repos, but probably even to the `docs` repo). But that's up to the docs team.